<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDialog</class>
 <widget class="QDialog" name="CSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>226</width>
     <height>188</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>128</y>
      <width>202</width>
      <height>28</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="heightSpinBox">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>44</y>
      <width>147</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="pestsSpinBox">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>100</y>
      <width>147</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="widthSpinBox">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>72</y>
      <width>147</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>44</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>72</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>100</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pests</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>14</y>
      <width>147</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Perferences</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
